client/my-sites/current-site/notice.jsx

```diff
@@ -114,10 +114,16 @@ export class SiteNotice extends React.Component {
 
 		const nonWPCOMDomains = reject(
 			this.props.domains,
-			domain => domain.isWPCOMDomain || domain.name.endsWith( '.wpcomstaging.com' )
+			domain =>
+				domain.isWPCOMDomain ||
+				domain.name.endsWith( '.wpcomstaging.com' ) ||
+				( domain.registrationDate &&
```

Comment:
Perhaps we should make the registration date check separate from the domains check to preserve the meaning of the `nonWPCOMDomains` const.

Reply:
Good point - but I'd prefer not to do a separate run through the array, so I've just renamed the variable to `eligibleDomains`.

Reply:
Love it. 💙